tensorflow/compiler/xla/service/gpu/cudnn_conv_runner.cc

```diff
@@ -73,7 +73,7 @@ class ScratchBufAllocator : public se::ScratchAllocator {
   bool allocated_ = false;
 };
 
-template <typename T>
+template <typename T, typename BIAS_T>
```

Comment:
`BIAS_T` -> `BiasType` per Google C++: https://google.github.io/styleguide/cppguide.html#Type_Names

Reply:
I will make the change and bookmarked the guide.  Thank you.

Reply:
Done